tools/interactive/interactivetool_openrefine.xml

```diff
@@ -0,0 +1,64 @@
+<tool id="interactive_tool_openrefine" tool_type="interactive" name="openrefine" version="0.1">
+    <description>Working with messy data: cleaning it; transforming it from one format into another; and extending it with web services and external data.</description>
+    <requirements>
+        <container type="docker">ylebras/openrefine-docker</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="Openrefine visualisation" requires_domain="True">
+            <port>80</port>
+            <url></url>
+        </entry_point>
+    </entry_points>
+    <environment_variables>
+
+        <environment_variable name="HISTORY_ID" strip="True">${__app__.security.encode_id($outfile.history_id)}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
+        <environment_variable name="REMOTE_HOST">${__app__.config.galaxy_infrastructure_url}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
+        <environment_variable name="GALAXY_WEB_PORT">8080</environment_variable>
+        <environment_variable name="GALAXY_URL">${__app__.config.galaxy_infrastructure_url}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
+        <environment_variable name="API_KEY" strip="True">
```

Comment:
@yvanlebras there is a new method to do this, please check out the new juypter one.

Reply:
https://github.com/usegalaxy-eu/galaxy/blob/release_19.09_europe/tools/interactive/interactivetool_jupyter_notebook.xml#L15